wcomponents-core/src/main/java/com/github/bordertech/wcomponents/container/WrongStepAjaxInterceptor.java

```diff
@@ -170,7 +175,7 @@ private void handleRenderRedirect(final PrintWriter writer) {
 		writer.print("<ui:ajaxtarget id=\"" + triggerId + "\" action=\"replace\">");
 
 		// Redirect URL
-		writer.print("<ui:redirect url=\"" + redirectUrl + "\" />");
+		writer.print(String.format("<%s url=\"%s\"></%s>", TAG_REDIRECT, redirectUrl, TAG_REDIRECT));
```

Comment:
We don't repeat args to `String.format` and similar interpolation methods. There is always a way to reuse the argument again in a different place. I'll let you work out how 😉

Reply:
Ok that works but can you do it while still using `String.format` and without repeating the args?

Clue: read about [argument_index](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#syntax)